cmd/jaeger/internal/extension/remotesampling/config.go

```diff
@@ -78,10 +75,6 @@ func (cfg *Config) Unmarshal(conf *confmap.Conf) error {
 }
 
 func (cfg *Config) Validate() error {
-	if cfg.File == nil && cfg.Adaptive == nil {
```

Comment:
allowing empty file name does not mean we allow not specifying a provider

Reply:
Okay. so what shall we take as the default strategy, (since if we want to remove dependancy on sampling-strategies.json, then there would be no default input left). I had allowed null for that reason

Reply:
Do not confuse strategy (the final output) with the provider (file or adaptive). I still want to use file provider in this case, but without a file name the provider can return a default strategy, which is probabilistic/1.0

Reply:
The reason to still use the file provider is because the user can override the file name via `--set` CLI flag, even if they do not provide a full config file.

Reply:
okay, got it, so I just have to add this test back again.
I had accidentally closed this pr, so creating a new one